d2l-icon-link.html

```diff
@@ -1,23 +1,20 @@
 <link rel="import" href="../polymer/polymer.html">
-<link rel="import" href="../d2l-offscreen/d2l-offscreen-shared-styles.html">
 <link rel="import" href="d2l-icon.html">
 <link rel="import" href="d2l-icon-button-shared-styles.html">
 <link rel="import" href="d2l-icon-button-behavior.html">
 
```

Comment:
A bunch of problems here I think: a lot of the attributes that would have been on the old `<a>` element aren't getting moved over to the anchor -- `href`, `onclick`, etc. These will need to be transferred.